purchase_order_adj_asx/views/purchase_order_views.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='origin']" position="after">
+                <label for="evidence_status" string="Evidence Status"/>
+                <div>
+                    <field name="evidence_status"/>
+                    <div>
+                        <field name="evidence_text"/>
+                    </div>
+                </div>
+            </xpath>
+            <xpath expr="//field[@name='notes']" position="after">
+                <field name="memo" class="oe_inline" placeholder="Internal Memo..."/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="purchase_order_tree" model="ir.ui.view">
+        <field name="name">purchase.order.tree</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='origin']" position="after">
+                <field name="memo"/>
+                <field name="evidence_status"/>
+                <field name="evidence_text"/>
+                <field name="message_attachment_count" string="Attachments"/>
+                <button name="action_get_attachment_view" string="View Attachments" type="object" icon="fa-paperclip"/>
```

Comment:
@SokreamPhanqrl Probably better to remove the strings from these fields, following the style of expense sheet.  What do you think?

Reply:
It's okay if user knows what those number are about , but what if they don't(for the first time). But anyway, it is better to remove it. :D